zstacklib/zstacklib/utils/linux.py

```
@@ -3152,11 +3176,20 @@ def file_has_config(fpath):
+                     '/var/run/virtqemud.pid', '/run/virtqemud.pid']:
+        if not os.path.exists(pid_file):
+            continue
 
-    with open('/var/run/libvirtd.pid') as f:
-        return int(f.read())
+        try:
+            with open(pid_file) as f:
+                pid = int(f.read())
+            if process_exists(pid):
+                return pid
+        except:
```

Ruff (0.15.15)

[error] Do not use bare `except`

(E722)